README.md

```diff
@@ -17,7 +17,7 @@ You can install the plugin by adding a config to `.tflint.hcl` and running `tfli
 ```hcl
 plugin "azurerm" {
     enabled = true
-    version = "0.17.1"
+    version = "0.17.2"
```

Comment:
Not sure what is the policy here, but if there's a new rule perhaps 0.18 is better?

Reply:
@wata727, moreover should I bump the version as part of the PR at all or is there another process for that?
I've noticed also that there's a changelog..

Reply:
Version and changelog changes are usually done separately before release. No changes are required here.